src/Lorentz/Contracts/BaseDAO/TZIP16Metadata.hs

```diff
@@ -30,17 +38,86 @@ knownBaseDAOMetadata = mconcat
   , views baseDAOViews
   ]
 
+------------------------------------------------------------------------
+-- Off-chain views
+------------------------------------------------------------------------
+
+type DaoView = View (ToT MetadataStorage)
+
 baseDAOViews :: [View $ ToT MetadataStorage]
```

Comment:
`DaoView` can be used here.

Reply:
Yep, noticed this later, though with the following commits these types are not so same anymore.